adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -1172,37 +1172,33 @@ class ADAMContext private (@transient val sc: SparkContext) extends Serializable
   /**
    * Loads variant annotations stored in VCF format.
    *
-   * @see loadVcf
-   * @see loadVariantAnnotations
-   *
    * @param filePath The path to the VCF file(s) to load annotations from.
-   * @return Returns DatabaseVariantAnnotationRDD.
+   * @return Returns VariantAnnotationRDD.
    */
-  def loadVcfAnnotations(filePath: String): DatabaseVariantAnnotationRDD = {
-    loadVcf(filePath).toDatabaseVariantAnnotationRDD
+  def loadVcfAnnotations(
+    filePath: String): VariantAnnotationRDD = {
+    loadVcf(filePath).toVariantAnnotationRDD
```

Comment:
Just for tracking RE @jpdna's [comment about not getting any annotations from a VCF](https://github.com/bigdatagenomics/adam/pull/1250#issuecomment-260020874), this line is the culprit. Specifically, `loadVcf` right now just parses the `Genotype`s. We should make the `VariantContextConverter` parse out the annotations by default in the follow on PR.